app/selenium_ui/jsm/pages/agent_pages.py

```diff
@@ -47,8 +47,14 @@ def get_app_version(self):
         return text.split('#')[0].replace('(v', '')
 
     def get_node_id(self):
-        text = self.__get_footer_text()
-        return text.split(':')[-1].replace(')', '')
+        text = self.get_element(LoginPageLocators.footer).text
+        text_split = text.split(':')
+        if len(text_split) == 2:
+            return "SERVER"
+        elif len(text_split) == 3:
+            return text_split[2].replace(')', '')
+        else:
+            return f"Warning:  failed to get the node id from {text}"
```

Comment:
"Let's change text to f"Warning: failed to get the node information from '{text}'." - this way we could easily catch a case when text is an empty string."

Probably, you should change it here too